packages/iov-bns/src/decode.ts

```diff
@@ -97,7 +97,7 @@ export function ensure<T>(maybe: T | null | undefined, msg?: string): T {
   return maybe;
 }
 
-function decodeNumericId(id: Uint8Array): number {
+export function decodeNumericId(id: Uint8Array): number {
```

Comment:
For escrow IDs I suggest using an implementation that ensures 8 byte inputs, like `Uint64.fromBytesBigEndian(id).toNumber()`. Maybe we want a dedicated function for that, depending on how `decodeNumericId` is used

Reply:
Did you delete your last comment?

Versions are 4 byte values as far as I know

Reply:
Ah sorry, not versions, but versioned IDs.